server/src/db/seeders/UserSeeder.js

```diff
@@ -0,0 +1,32 @@
+import { User } from "../../models/index.js";
+
+class UserSeeder {
+  static async seed() {
+    const usersData = [
+      {
+        email: "tyler@gmail.com",
+        username: "Tyler",
+        cryptedPassword: "123",
```

Comment:
if you want to be able to log in as these seeded users, you will need to set up your users to be created with a `password`. The User model will handle encrypting the value for you and set the `cryptedPassword` when it saves to the database